sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/HttpFaultInjectingTests.java

```diff
@@ -112,13 +115,13 @@ private HttpClient getFaultInjectingWrappedHttpClient() {
         switch (ENVIRONMENT.getHttpClientType()) {
             case NETTY:
                 return HttpClient.createDefault(new HttpClientOptions()
-                    .readTimeout(Duration.ofSeconds(5))
-                    .responseTimeout(Duration.ofSeconds(5))
+                    .readTimeout(Duration.ofSeconds(2))
```

Comment:
In theory the timeout could be shortened further but 2 seconds seems reasonable to prevent cases where the service response is slow being confused with fault injection.